nautobot/core/cli/__init__.py

```diff
@@ -115,6 +115,7 @@ def _configure_settings(config):
     os.makedirs(settings.MEDIA_ROOT, exist_ok=True)
     os.makedirs(os.path.join(settings.MEDIA_ROOT, "devicetype-images"), exist_ok=True)
     os.makedirs(os.path.join(settings.MEDIA_ROOT, "image-attachments"), exist_ok=True)
+    os.makedirs(os.path.join(settings.NAUTOBOT_UI_DIR, "build", "static"), exist_ok=True)
     os.makedirs(settings.STATIC_ROOT, exist_ok=True)
```

Comment:
Looks solid,  but I wonder if instead we should do something like:

```suggestion
    for directory in settings.STATICFILES_DIRS:
        os.makedirs(directory, exist_ok=True)
```

What do you think?

Reply:
Oh wait, I'm confusing myself. `STATIC_ROOT` is a destination directory, while `NAUTOBOT_UI_DIR/build/static` is a source directory (but only if we've run `nautobot-server build_ui`). So the current implementation is correct I think.

Reply:
Confirmed locally that the correct dir is being created.